ros2_java_android.repos

```diff
@@ -2,80 +2,84 @@ repositories:
   eProsima/Fast-CDR:
     type: git
     url: https://github.com/eProsima/Fast-CDR.git
-    version: v1.0.7
+    version: release/1.0.11
   eProsima/Fast-RTPS:
     type: git
     url: https://github.com/eProsima/Fast-RTPS.git
-    version: 7a0b0fe7ca8d2c4ea41e36744c6024c263a6505a
+    version: release/1.8.2
```

Comment:
Similar to my other comment, I think this should be `v1.8.2`.

Reply:
I didn't get your new comment.
It already v1.8.2
and it already v1.0.11
Can you check again?

Reply:
I'm suggesting changing the version to be the git tag (`v1.8.2`). like it is in the ROS 2 repos file, instead of the branch (`release/1.8.2`).

Reply:
Now I understand, and I change accordingly.